crates/fluvio-sc/Cargo.toml

```diff
@@ -66,6 +66,7 @@ fluvio-protocol = { workspace = true }
 fluvio-socket = { workspace = true }
 fluvio-service = { workspace = true  }
 flv-tls-proxy = { workspace = true }
+fluvio = { workspace = true }
```

Comment:
This bring too much dependency as well.  Only need `admin` feature flag.    May need to update fluvio library to properly config `admin` feature flag